CIScripts/Test/TestRunner.ps1

```diff
@@ -103,3 +103,36 @@ function Run-Tests {
         }
     })
 }
+
+function Run-Tests {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+
+    try {
+        Run-TestsWithoutLogs -Sessions $Sessions
```

Comment:
Maybe it would be better to add this try { } catch where Run-Tests was originally called, and in catch we could handle using separate method.

In that case there will be no need for additional method Run-TestsWithoutLogs - which actually creates some logs.

What do you think about that ?